src/PaymentGateways/PayPalCommerce/onBoardingRedirectHandler.php

```diff
@@ -390,41 +390,17 @@ private function isAdminSuccessfullyOnBoarded($merchantId, $accessToken, $usesCu
         }
 
         if (! $onBoardedData['primary_email_confirmed']) {
-            $errorMessage[] = esc_html__('Confirm your primary email address', 'give');
-        }
-
-        if (! $usesCustomPayments) {
-            return count($errorMessages) > 1 ? $errorMessages : true;
-        }
-
-        if (array_diff(['products', 'capabilities'], array_keys($onBoardedData))) {
-            $errorMessages[] = esc_html__(
-                'Your account was expected to be able to accept custom payments, but is not. Please make sure your
-				account country matches the country setting. If the problem persists, please contact PayPal.',
-                'give'
-            );
-
-            // Return here since the rest of the validations will definitely fail
-            return $errorMessages;
-        }
-
-        // Grab the PPCP_CUSTOM product from the status data
-        $customProduct = current(
-            array_filter(
-                $onBoardedData['products'],
-                function ($product) {
-                    return $product['name'] === 'PPCP_CUSTOM';
-                }
-            )
-        );
-
-        if (empty($customProduct) || $customProduct['vetting_status'] !== 'SUBSCRIBED') {
-            $errorMessages[] = esc_html__('Reach out to PayPal to enable PPCP_CUSTOM for your account', 'give');
+            $errorMessages[] = esc_html__('Confirm your primary email address', 'give');
         }
 
         // Loop through the capabilities and see if any are not active
+        $requiredCapabilities = ['CUSTOM_CARD_PROCESSING'];
```

Comment:
@ravinderk From the doc:
> In the Show Seller Status call, the primary things your integration should 
check for is:
> 1. “payments_receivable”: “true”
> 2. “primary_email_confirmed”: “true”

Reply:
@JasonTheAdams Can I ignore this?

<img width="619" alt="image" src="https://user-images.githubusercontent.com/1784821/232859627-2cf9fc90-8c8f-4bea-92a2-52e1ebf3c77b.png">

It would be better to keep this. Let me know if you want me to remove it.